app/overrides/decidim/consultations/admin/responses/add_blank_vote_to_consultations_responses_form.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Deface::Override.new(virtual_path: 'decidim/consultations/admin/responses/_form',
+                     name: 'add_blank_vote_to_consultations_responses_form',
+                     insert_before: "erb[loud]:contains('form.translated :text_field')",
+                     text: "
+                        <% if current_question.blank_vote %>
+                          <%= form.check_box :blank_vote, { onchange: 'blankVote(this)' }, { data: { 'es': I18n.with_locale(:es) { t('activemodel.attributes.response.blank_vote') },
```

Comment:
I think is better to not consider the available languages like something that will never change. As I see it, it is better to collect the premises directly from the organization and then their corresponding values to assign the blank voting label.

`
<%= form.check_box :blank_vote, { onchange: 'blankVote(this)' }, { data: 
      Hash[*current_question.organization.available_locales.map{|locale| 
         [locale, I18n.with_locale(locale.to_sym) { t('activemodel.attributes.response.blank_vote') }] 
      }.flatten] }.to_json 
%>
`